feflow/protocols/nonequilibrium_cycling.py

```diff
@@ -997,6 +997,8 @@ def _create(
         if extends:
             raise NotImplementedError("Can't extend simulations yet")
 
+        mapping = mapping[0] if isinstance(mapping, list) else mapping  # type: ignore
```

Comment:
Doing this silently is, in my opinion, a really bad idea when it comes to the user experience. I.e. imagine passing in a list of mappings hoping you'd get multiple transformations, and then you end up with just one (and you're not even sure which because you didn't get told!).

If we're doing this then we should put in something that raises an error if the list is > 1. That's what we do in `openfe` - and short-medium term I want to spend time to merge the two protocol's setups (adding in all the validation from openfe), but in a more immediate case it would be best to just have the one line here.

Reply:
Agreed! Would you recommend just using the openfe [validate function ](https://github.com/OpenFreeEnergy/openfe/blob/f4475bfe47fa53614cf22ba7e7662a6cf288ba4b/openfe/protocols/openmm_rfe/equil_rfe_methods.py#L176)rather than writing something similar again?

Reply:
went with the validate method in [82447f0](https://github.com/OpenFreeEnergy/feflow/pull/90/commits/82447f0b0b75780ea422b6ac0a52fedd948a3cfd)

Reply:
I think that should work, we probably should migrate the `_validate_alchemical_components` into `FEFLow`, or at least put a note somewhere that we expect the validation to be migrated here in the future.

I think this is very related to #55 but it still needs some work to finish it.

Reply:
I would go with the note personally, I woulld prefer spend energies towards merging the setups for the two protocols rather than doing lots of workarounds until we do.